uk_election_timetables/elections/city_of_london_local.py

```diff
@@ -0,0 +1,40 @@
+from datetime import date
+
+from uk_election_timetables.calendars import working_days_before, Country
+from uk_election_timetables.election import Election
+
+
+"""
+TODO:
+This is insufficently nuanced
+https://github.com/DemocracyClub/uk-election-timetables/issues/8
+"""
+
+
+class CityOfLondonLocalElection(Election):
+    def __init__(self, poll_date: date):
+        """
+        :param poll_date: a datetime representing the date of the poll
+        """
+        Election.__init__(self, poll_date, Country.ENGLAND)
+
+    @property
+    def sopn_publish_date(self) -> date:
+        """
+        Calculate the SOPN publish date for a City of London local election.
+
+        :return: a datetime representing the expected publish date
+        """
+
+        return working_days_before(self.poll_date, 17, super()._calendar())
```

Comment:
This is the right number of days for City of London, but its the wrong calculation (see https://github.com/DemocracyClub/uk-election-timetables/issues/8#issuecomment-749574325 ). I haven't bothered to try and encode the additional rules yet.

For this reason, I've not bothered writing tests for this function. Lets get it right first then nail it down with tests.